tests/components/govee_light_local/test_config_flow.py

```diff
@@ -21,6 +20,9 @@ async def test_creating_entry_has_no_devices(
     with patch(
         "homeassistant.components.govee_light_local.config_flow.GoveeController",
         return_value=mock_govee_api,
+    ), patch(
+        "homeassistant.components.govee_light_local.config_flow.DISCOVERY_TIMEOUT",
+        0,
     ):
```

Comment:
An alternative would be to patch `asyncio.timeout` so that it raises `TimeoutError`, as is done in the light test.

Reply:
I don't think patching a generic is a better alternative TBH